component/core/src/test/java/org/exoplatform/social/core/listeners/ActivityAttachmentMetadataListenerTest.java

```diff
@@ -0,0 +1,165 @@
+ * Copyright (C) 2020 - 2023 Meeds Association contact@meeds.io
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package org.exoplatform.social.core.listeners;
+
+import org.exoplatform.container.xml.InitParams;
+import org.exoplatform.container.xml.ObjectParameter;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.exoplatform.social.common.ObjectAlreadyExistsException;
+import org.exoplatform.social.core.activity.model.ExoSocialActivity;
+import org.exoplatform.social.core.activity.model.ExoSocialActivityImpl;
+import org.exoplatform.social.core.identity.model.Identity;
+import org.exoplatform.social.core.identity.provider.SpaceIdentityProvider;
+import org.exoplatform.social.core.jpa.storage.dao.jpa.MetadataDAO;
+import org.exoplatform.social.core.manager.ActivityManager;
+import org.exoplatform.social.core.manager.IdentityManager;
+import org.exoplatform.social.core.space.model.Space;
+import org.exoplatform.social.core.test.AbstractCoreTest;
+import org.exoplatform.social.metadata.MetadataService;
+import org.exoplatform.social.metadata.MetadataTypePlugin;
+import org.exoplatform.social.metadata.model.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ActivityAttachmentMetadataListenerTest extends AbstractCoreTest {
+
+  private static final Log        LOG                = ExoLogger.getLogger(ActivityAttachmentMetadataListenerTest.class);
+
+  private static final String     METADATA_TYPE_NAME = "attachments";
+
+  private static final Long       FILE_ID            = 12l;
+
+  private static final Long       AUDIENCE_ID        = 5l;
+
+  private static final String     PARENT_OBJECT_ID   = "7";
+
+  private Identity                johnIdentity;
+
+  private ActivityManager         activityManager;
+
+  private IdentityManager         identityManager;
+
+  private MetadataService         metadataService;
+
+  private MetadataDAO             metadataDAO;
+
+  private List<ExoSocialActivity> tearDownActivityList;
+
+  private List<Space>             tearDownSpaceList;
+
+  @Override
+  public void setUp() throws Exception {
+    try {
+      super.setUp();
+    } catch (Exception e) {
+      LOG.error("Error initializing parent Test class", e);
+    }
```

Comment:
No try/catch in Test classes please
 
```suggestion
      super.setUp();
```

Reply:
no need since I removed ActivityAttachmentMetadataListener file!